src/panels/config/info/system-health-card.ts

```diff
@@ -128,14 +128,24 @@ class SystemHealthCard extends LitElement {
   }
 
   private _copyInfo(): void {
+    // We have to copy title text and content separately, because
+    // copying the whole <ha-card> would also copy the tooltip text.
     const selection = window.getSelection()!;
     selection.removeAllRanges();
 
-    const copyElement = this.shadowRoot?.querySelector(
-      "ha-card"
+    let copyElement = this.shadowRoot?.querySelector(
+      ".card-header-text"
     ) as HTMLElement;
 
-    const range = document.createRange();
+    let range = document.createRange();
+    range.selectNodeContents(copyElement);
+    selection.addRange(range);
+
+    copyElement = this.shadowRoot?.querySelector(
+      ".card-content"
+    ) as HTMLElement;
+
+    range = document.createRange();
```

Comment:
Do we need to create a new range?

Reply:
Yes, if I reuse the existing range, then the content from the card-header is replaced by the card-content and therefore not copied.